ICAFIX/scripts/functionhighpassandvariancenormalize.m

```diff
@@ -192,29 +225,32 @@ function functionhighpassandvariancenormalize(TR,hp,fmri,WBC,varargin)
 
 end
 
-
-% NOTE: To interpret the logic of the following "Compute, Save, Apply" sections need to know that this function expects that: 
-% (1) individual run time series will be passed in with hp requested
-% (2) concatenated (multi-run) time series will be passed in with hp<0
-%% [This is a bit of hack: would be cleaner if we disentangled the operation of this function from the specific needs of 'hcp_fix_multi_run'].
+if VNhalfdim
+    if dovol
+        VN = ceil(size(cts, 2) / 2);
+    else
+        VN = ceil(size(B0.cdata, 2) / 2);
+    end
+end
 
 %% Compute VN map
-if hp>=0
-    if dovol > 0
-        Outcts=icaDim(cts,0,1,-1,volhpwishart); %0=Don't detrend, 1=Initialize variance normalization at 1, -1=Converge with running dim average, Volume fits two distributions to deal with MNI transform 
+if singlerun
+    if dovol
+        Outcts = icaDim(cts, 0, VN, iters, volwishart); %0=Don't detrend, 1=Initialize variance normalization at 1, -1=Converge with running dim average, Volume fits 2 distributions by default to deal with MNI transform
     end
     
-    OutBO=icaDim(BO.cdata,0,1,-1,3); %0=Don't detrend, 1=Initialize variance normalization at 1, -1=Converge with running dim average, CIFTI fits three distributions to deal with volume to CIFTI mapping
+    OutBO = icaDim(BO.cdata, 0, VN, iters, ciftiwishart); %0=Don't detrend, 1=Initialize variance normalization at 1, -1=Converge with running dim average, CIFTI fits 3 distributions by default to deal with volume to CIFTI mapping
```

Comment:
Very ambiguous what variable the comment is referring to here.  Is it the second argument ("0"), or the 3rd argument ("VN")?

Reply:
And looking at `icaDim.m` doesn't help, because `icaDim.m` has no usage/help information at all.

Reply:
The values in the comment are the arguments in order, excluding the input data.  Some still contain the previous hardcoded values rather than explicitly saying they are the defaults of what are now variables.

Reply:
That's very confusing -- no one is going to get that. I think users would be much better served by simply adding some usage/functionality info into `icaDim.m` itself.